### ISSUE TITLE
BERT4Rec模型在使用Yelp数据集进行训练时，一直停在53%不动，仿佛代码运行过程中陷入了死循环

### ISSUE THREAD

**Issue by Figfavture** [RUCAIBox/RecBole#1156]
**描述这个 bug**
BERT4Rec模型在使用Yelp数据集进行训练时，一直停在53%不动，仿佛代码运行过程中陷入了死循环

**如何复现**
复现这个 bug 的步骤：
1. 您引入的额外 yaml 文件yelp：
```shell
###sequential
# dataset config
field_separator: "\t"
seq_separator: " "
USER_ID_FIELD: user_id
ITEM_ID_FIELD: business_id
RATING_FIELD: stars
TIME_FIELD: date
NEG_PREFIX: neg_
ITEM_LIST_LENGTH_FIELD: item_length
LIST_SUFFIX: _list
MAX_ITEM_LIST_LENGTH: 100
POSITION_FIELD: position_id
load_col:
  inter: [user_id, business_id, stars, date]
lowest_val:
  stars: 3
drop_filter_field: False

min_user_inter_num: 15
min_item_inter_num: 5
```
**预期**
运行模型的时候训练进度会卡住不动

**屏幕截图**
添加屏幕截图以帮助解释您的问题。（可选）

**实验环境：**
 - 操作系统: windows
- RecBole 版本 :1.0.0
 - Python 版本:3.8.3
- PyTorch 版本:1.8.1
- cudatoolkit 版本 :10.1

**Comment by chenyuwuxin** [comment]
@Figfavture 感谢您的关注！
你的配置文件存在错误，我们最新的版本中不再会使用类似`min_item_inter_num`以及`min_user_inter_num`这类字段，请参考最新的项目文档[Docs](https://recbole.io/docs/user_guide/config/data_settings.html?highlight=user_inter_num_interval)，并且确保使用的是v1.0.0版本的RecBole。此外，BERT4Rec的一个示例配置文件如下：
```
# dataset config
field_separator: "\t"
seq_separator: " "
USER_ID_FIELD: user_id
ITEM_ID_FIELD: business_id
RATING_FIELD: stars
TIME_FIELD: date
NEG_PREFIX: neg_
ITEM_LIST_LENGTH_FIELD: item_length
LIST_SUFFIX: _list
MAX_ITEM_LIST_LENGTH: 50
POSITION_FIELD: position_id
load_col:
  inter: [user_id, business_id, stars, date]

user_inter_num_interval: "[15,inf)"
item_inter_num_interval: "[15,inf)"
val_interval:
  stars: "[3,inf)"


# training and evaluation
epochs: 500
train_batch_size: 4096
eval_batch_size: 2000
valid_metric: MRR@10
eval_args:
  split: {'LS':"valid_and_test"}
  order: TO
neg_sampling: ~

# model
embedding_size: 64 
```

**Comment by Figfavture** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    非常感谢您的回复。
        
    
        
            
        
    
    
     
    
        
                    ***@***.***
        
    
    
        签名由网易邮箱大师定制
    
 

    在2022年2月24日 15:14，Tian ***@***.***> 写道： 




@Figfavture 感谢您的关注！

你的配置文件存在错误，我们最新的版本中不再会使用类似min_item_inter_num以及min_user_inter_num这类字段，请参考最新的项目文档Docs，并且确保使用的是v1.0.0版本的RecBole。此外，BERT4Rec的一个示例配置文件如下：

# dataset config

field_separator: "\t"

seq_separator: " "

USER_ID_FIELD: user_id

ITEM_ID_FIELD: business_id

RATING_FIELD: stars

TIME_FIELD: date

NEG_PREFIX: neg_

ITEM_LIST_LENGTH_FIELD: item_length

LIST_SUFFIX: _list

MAX_ITEM_LIST_LENGTH: 50

POSITION_FIELD: position_id

load_col:

  inter: [user_id, business_id, stars, date]



user_inter_num_interval: "[15,inf)"

item_inter_num_interval: "[15,inf)"

val_interval:

  stars: "[3,inf)"





# training and evaluation

epochs: 500

train_batch_size: 4096

eval_batch_size: 2000

valid_metric: ***@***.***

eval_args:

  split: {'LS':"valid_and_test"}

  order: TO

neg_sampling: ~



# model

embedding_size: 64
